### PR TITLE
Add Codecov to coverage docs

### PR DESCRIPTION
**Pull request by stevepeak**
Just had a chat with Kunal Jain and previously @bellkev about adding Codecov docs.

Thank you and let me know if you would like any modifications.

### CODE REVIEW

**Comment by felicianotech** [comment]
Thank you @stevepeak, will take a look.

File: jekyll/_docs/code-coverage.md [hunk 1]
```diff
@@ -94,6 +94,24 @@ you can see beautifully rendered HTML in the UI.
...
 
+### Codecov
+If you're a [Codecov](https://codecov.io?src=circleci-docs) customer, integration with CircleCI can be as easy as
+
+```yaml
+test:
+  post:
+    - bash <(curl -s https://codecov.io/bash)
+```
+> No upload token is requires for open source repositories on GitHub.
```
Comment (felicianotech) [inline jekyll/_docs/code-coverage.md]: "requires" -> required?

**Comment by felicianotech** [comment]
Hey @stevepeak, just that one grammar issue and this is good to go.

**Comment by stevepeak** [comment]
@felicianotech 👍 thank you!

**Comment by felicianotech** [comment]
Looks great @stevepeak, thank you. I hope users will get value out of this.

The way you guys have example repos setup is nice. Works well. :)

**Comment by stevepeak** [comment]
Thanks @felicianotech 👍

**Pull request merged**